addOns/llm/src/main/java/org/zaproxy/addon/llm/communication/Confidence.java

```diff
@@ -1,10 +1,30 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.zaproxy.addon.llm.communication;
 
 import dev.langchain4j.model.output.structured.Description;
 
 public class Confidence {
 
-    @Description("The level of confidence, typically represented as a percentage or a descriptive term")
+    @Description(
+            "The level of confidence, typically represented as a percentage or a descriptive term")
```

Comment:
I'm not familiar with this annotation, do these values endup being shown in the GUI anywhere or are these just internal "tips"? (For lack of a better term)

Reply:
No they won't endup in the UI, It's part of the langchain4j library, and they are use to explain to the LLM what output need to be included in the response.